ws-server/Cargo.toml

```diff
@@ -12,7 +12,7 @@ documentation = "https://docs.rs/jsonrpsee-ws-server"
 [dependencies]
 thiserror = "1"
 futures-channel = "0.3.14"
-futures-util = { version = "0.3.14", default-features = false, features = ["io"] }
+futures-util = { version = "0.3.14", default-features = false, features = ["io", "async-await-macro"] }
```

Comment:
Nit: It doesn't look like we've added any code that requires this feature; do we want it?

Reply:
it was introduced in another PR, thus doesn't compile standalone such as if you `cd ws-server && cargo check` outside the workspace it doesn't work. Because, features in Rust are additive (the union of all features) so if another crate adds that feature it's enabled for all in workspace if that makes sense.

TLDR; on master `cd ws-server && cargo check` doesn't work but `cargo build -p ws-server` works :P

Reply:
Aah gotcha! Thanks for the explanation!